pandas/tests/test_algos.py

```diff
@@ -248,6 +248,12 @@ def test_uint64_factorize(self):
         tm.assert_numpy_array_equal(labels, exp_labels)
         tm.assert_numpy_array_equal(uniques, exp_uniques)
 
+    def test_deprecate_order(self):
+        data = np.array([2**63, 1, 2**63], dtype=np.uint64)
```

Comment:
can you add the gh issue number here as a comment

Reply:
Good point.  I added an additional comment so we can remember to deprecate the test once the ``order`` keyword is finally removed.